### PR TITLE
issue: #40373 default setting for panel location

### PR DESCRIPTION
**Pull request by pfongkye**

### CODE REVIEW

File: src/vs/workbench/electron-browser/main.contribution.ts [hunk 1]
```diff
@@ -216,6 +216,12 @@ let workbenchProperties: { [path: string]: IJSONSchema; } = {
 		'default': 'left',
 		'description': nls.localize('sideBarLocation', "Controls the location of the sidebar. It can either show on the left or right of the workbench.")
 	},
+	'workbench.panel.defaultLocation': {
+		'type': 'string',
+		'enum': ['bottom', 'right'],
+		'default': 'bottom',
+		'description': nls.localize('panelDefaultLocation', "Controls the location of the panel. It can either show at the bottom or on the right of the workbench.")
```
Comment (isidorn) [inline src/vs/workbench/electron-browser/main.contribution.ts]: I think this controls the default location, so I would prefer the folowing
`"Control the default location of the panel."

File: src/vs/platform/telemetry/common/telemetryUtils.ts [hunk 2]
```diff
@@ -153,6 +153,7 @@ const configurationValueWhitelist = [
 	'php.validate.run',
 	'workbench.welcome.enabled',
 	'workbench.startupEditor',
+	'workbench.panel.defaultLocation',
```
Comment (isidorn) [inline src/vs/platform/telemetry/common/telemetryUtils.ts]: This is not needed imho, please remove it.

**Comment by isidorn** [comment]
@pfongkye I have added 2 minor comments. Please address them and merge latest master into your PR, resolve conflicts and we can merge this in.

Thanks a lot 🍻

**Comment by pfongkye** [comment]
@isidorn I made the changes and merged master. Thanks for the review 👍

**Comment by isidorn** [comment]
@pfongkye this looks great now, thanks a lot I merged it in 🍻

**Pull request merged**